### ISSUE TITLE
Refact: maybe the context is a little complex

### ISSUE THREAD

**Issue by cmsax** [cmsax/linka#13]
Let's talk about pros and cons about Context and redux like state management tools

**Comment by use-the-fork** [comment]
I agree. I'm not sure Redux is the right way to go since we would lose the flexibility of putting anything in the drawer but I can bring the context down to just one that controls the Toast, Drawer, and progress loader. Let me know what direction you want to go in for this.
